src/EditorFeatures/CSharpTest/AddBraces/AddBracesTests.cs

```diff
@@ -615,7 +1100,21 @@ static void Main()
                 return;
             }
     }
-}");
+}",
+                (PreferBracesPreference)bracesPreference,
+                expectDiagnostic);
+        }
+
+        private async Task TestMissingOrInRegularAndScriptAsync(string initialMarkup, string expectedMarkup, PreferBracesPreference bracesPreference, bool expectDiagnostic)
```

Comment:
i would consider just calling this "TestAsync".  The "TestMissing..." makes it strongly sound like no result is expected.